c10/core/TensorImpl.h

```diff
@@ -613,10 +616,9 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
   }
 
   IntArrayRef sizes_default() const {
-    // TODO: force backtrace to be printed on this error
-    TORCH_CHECK(
-        !has_symbolic_sizes_strides_,
-        "Cannot call sizes() on tensor with symbolic sizes/strides");
+    if (C10_UNLIKELY(has_symbolic_sizes_strides_)) {
+      throw_cannot_call_with_symbolic("sizes");
```

Comment:
What's the issue with leaving it inline - is the issue that once you update the error check to grab a backtrace, then leaving it inline in the header would make the binary size of `sizes_default` a lot bigger?

Reply:
Actually, this is mostly a hygiene thing. Exception throwing is heavy, don't inline it lol

Reply:
I was going to make this call some special new codepath that would have made binary size bigger but in the end I did it another way.